src/settings/AudioSettings.tsx

```diff
@@ -200,10 +229,37 @@ export default function GeneralSettings(props: ISettingsPanelProps) {
           </IconButton>
         </Tooltip>
       </Box>
+      <Box component="span" sx={{ display: 'flex', alignItems: 'center' }}>
+        <FormControlLabel
+          control={
+            <Switch
```

Comment:
I really like this! I've used checkboxes on the other pages, I'm inclined to switch them all to "Switches", looks way better.